bootstrap/kubeadm/main.go

```diff
@@ -151,6 +153,12 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&webhookCertDir, "webhook-cert-dir", "/tmp/k8s-webhook-server/serving-certs/",
 		"Webhook cert dir, only used when webhook-port is specified.")
```

Comment:
```suggestion
		"Webhook cert dir.")
```
I don't think this is true (anymore, but since ~ v1alpha3 when we dropped webhook system)

Reply:
done across the board

Reply:
Almost. Same for your new flags :0

Reply:
good catch, copy paste 😄  done!